src/main/java/net/sf/jabref/bibtex/BibEntryWriter.java

```diff
@@ -108,11 +108,11 @@ private void writeRequiredFieldsFirstRemainingFieldsSecond(BibEntry entry, Write
             }
         }
         for (String field : remainingFields) {
-            writeField(entry, out, field, indentation);
+            hasWritten = hasWritten | writeField(entry, out, field, hasWritten, indentation);
```

Comment:
Yeah, I removed it in https://github.com/JabRef/jabref/commit/d2db24e749b25babd3e854670f09cc7c9a9c0011. So probably a merge problem.